### PR TITLE
Fix issue 550

### PR DESCRIPTION
**Pull request by aorlati**
Several updates ij order to match Medicina configuration and to prepare the release to be installed in Medicina

### CODE REVIEW

File: SystemMake/Makefile [hunk 1]
```diff
@@ -120,7 +120,7 @@ ifeq ($(STATION),Medicina)
 		SlaLibrary IRALibrary TextWindowLibrary ParserLibrary ComponentProxy ModbusChannel XarcosLibrary \
 	    PyTestingLibrary \
 		AntennaBoss Observatory OTF PointingModel Refraction SkySource Moon FitsWriter Scheduler ReceiversBoss ExternalClients CalibrationTool \
-		TotalPower NoiseGenerator MedicinaMount MedWeatherStation CustomLogger XBackend MedicinaMinorServo\
+		TotalPower NoiseGenerator MedicinaMount MedcinaPyLocalOscillator MedicinaPyDMed CustomLogger XBackend MedicinaMinorServo\
```
Comment (giuseppe-carboni) [inline SystemMake/Makefile:123]: This might be a typo "MedcinaPy..."

File: Common/Servers/ReceiversBoss/src/RecvBossCore_mc.i [hunk 2]
```diff
@@ -1,10 +1,10 @@
 //#define RB_DEBUG
 
-#define RECV_ADDRESS "192.167.189.2"
+#define RECV_ADDRESS "192.168.51.2"
 #define RECV_PORT 2096
-#define FS_ADDRESS "192.167.189.62"
+#define FS_ADDRESS "192.168.51.14"
 #define FS_PORT 5002
-#define SXKL_ADDRESS "192.167.189.2"
```
Comment (giuseppe-carboni) [inline Common/Servers/ReceiversBoss/src/RecvBossCore_mc.i:7]: This code seems to be part of the Common branch, are we sure we want to keep some IP addresses in this part of the repository? This code might have to run in multiple stations

File: SystemMake/Makefile (hunk 1 above)
Comment (aorlati) [reply SystemMake/Makefile]: Typo fixed in lat commit

File: Common/Servers/ReceiversBoss/src/RecvBossCore_mc.i (hunk 2 above)
Comment (aorlati) [reply Common/Servers/ReceiversBoss/src/RecvBossCore_mc.i:7]: A switch at compile time avoids this piece of code is compiled for stations different from Medicina. 
Of course this should be included in a dedicated component.

**Review by giuseppe-carboni — APPROVED** [review]

**Pull request merged**